### ISSUE TITLE
Please use KICS 1.3.0 for the test

### ISSUE THREAD

**Issue by kaplanlior** [iacsecurity/tool-compare#14]
Thanks (:

**Comment by yi2020** [comment]
Can you open a PR to update the run_all_tools.sh script to use the right KICS version?

**Comment by kaplanlior** [comment]
You already run kics:latest (as all tools), so just rerun it and update the version used.

**Comment by yi2020** [comment]
Ah, I see it's a few days old. Did the following now:
* Cleared all kics_results.txt: `find . -name "kics_results.txt" -exec rm {} \;`
* Ran only the KICS portion of the `run_all_tools.sh` script (required update to use the "scan" command)
* Compared kics_results via PR #15 , found one test case that wasn't caught earlier. Updated its `results_summary.json` and re-ran README creator.

@kaplanlior can you take a look at the above-mentioned PR? Looks like there's one test case that is now caught by 1.3.0 and wasn't with the previous version. Any test cases I somehow missed? Are there test cases you'd like to add?

**Comment by ismailyenigul** [comment]
Hi @yi2020 
 I see some updates on checkov side as well. Can you run the tests on checkov latest.
Also It would be great if you can add Last Scan time to README.md

Thanks

**Comment by yi2020** [comment]
Yeah @ismailyenigul , can you open a separate issue for that and I'll handle it after this one? We need the PRs to be focused on just one tool at a time, otherwise they get really hard to follow.

**Comment by ismailyenigul** [comment]
Sure!

**Comment by kaplanlior** [comment]
> Ah, I see it's a few days old. Did the following now:
> 
> * Cleared all kics_results.txt: `find . -name "kics_results.txt" -exec rm {} \;`
> * Ran only the KICS portion of the `run_all_tools.sh` script (required update to use the "scan" command)
> * Compared kics_results via PR #15 , found one test case that wasn't caught earlier. Updated its `results_summary.json` and re-ran README creator.
> 
> @kaplanlior can you take a look at the above-mentioned PR? Looks like there's one test case that is now caught by 1.3.0 and wasn't with the previous version. Any test cases I somehow missed? Are there test cases you'd like to add?

Go ahead and merge it, thanks for the quick response. 
Our next version is going to have some good surprises regarding the results.
